app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/index.tsx

```diff
@@ -85,8 +86,22 @@ export const ChangeApprovalOperations: React.FC<
           className="bg-transparent text-success"
         >
           <p className="my-1" data-testid="in-review-label">
-            <Check className="align-top" /> All set! This experiment will{" "}
-            {actionDescription} as soon as it is approved.
+            Please ask someone on your team with review privileges, or a
+            qualified{" "}
+            <LinkExternal href={EXTERNAL_URLS.EXPERIMENTER_REVIEWERS}>
+              reviewer
+            </LinkExternal>
+            , to review and {actionDescription} your experiment.{" "}
+            <a
+              href="#copy"
+              onClick={(event) => {
+                event.preventDefault();
+                navigator.clipboard.writeText(window.location.toString());
```

Comment:
I feel like there's some potential sharing with the `CopyableURL` component if only to DRY up the `navigator.clipboard.writeText` bit but that component is currently fairly different than what you've got here so either way seems fine. You could copy the `className="cursor-copy"` that's over there though, and maybe consider making this a `span`?

The green text and blue links looks kind of strange to me but it follows the "success text" pattern so🤷‍♀️

Reply:
> The green text and blue links looks kind of strange to me

Agree, but not sure what the better alternative is here unless we update more than just this message style

> You could copy the className="cursor-copy" that's over there though, and maybe consider making this a span?

Added the class but leaving it as an anchor to maintain the style of a link (otherwise it's green/no underline)